test/clj/rems/browser_test_util.clj

```diff
@@ -174,21 +159,6 @@
         (log/warn e "WebDriver failed to start, retrying...")
         (run)))))
 
-(defn refresh-driver-fixture
```

Comment:
The reason for using this previously was that we can re-use the same browser as opening a new browser window / driver is much slower.

Seems like it was removed in https://github.com/CSCfi/rems/commit/dbe1dc49ed35f10e1806d4cb0564da1481da3f4b